src/Template/Adrs/xml/e2b.ctp

```diff
@@ -267,7 +271,7 @@
                 <drugrecurreadministration/>
                 <drugadditional/>
                 <activesubstance>
-                    <activesubstancename><?php echo $adrListOfDrug['drug_name']; ?></activesubstancename>
+                    <activesubstancename><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$adrListOfDrug['drug_name']); ?></activesubstancename>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate